README.md

```diff
@@ -115,7 +115,6 @@ In the example below, the manifest is located at `/build-aux/flatpak/org.gnome.z
         "--socket=fallback-x11",
         "--socket=wayland",
         "--filesystem=xdg-run/dconf",
-        "--filesystem=~/.config/dconf:ro",
         "--talk-name=ca.desrt.dconf",
         "--env=DCONF_USER_CONFIG_DIR=.config/dconf"
```

Comment:
if it is possible, please drop the remaining dconf mentions as they are no longer needed

Reply:
Sure